### ISSUE TITLE
Alpha-uppercase lists are copied as regular numbered lists when using the "paste" plugin.

### ISSUE THREAD

**Issue by niuage** [tinymce/tinymce#6024]
**What is the current behavior? Describe the bug**

tldr: look at this CodePen: https://codepen.io/niuage/pen/MWyVEpo

I want to paste an alpha-upper list from Google Docs into TinyMCE. It works without the "paste" plugin, doesn't work with it.

My question is: is there an option from this plugin that I'm missing that would fix this issue?

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

https://codepen.io/niuage/pen/MWyVEpo

**What is the expected behavior?**

The alpha-upper (and probably any other kind of non-numbered list) should be pasted properly, and not changed to a numbered list.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

TinyMCE: 5.4
Chrome.

**Comment by niuage** [comment]
No one?

**Comment by ashfordneil** [comment]
Hi!

Yes there is an option for this plugin that would fix this issue: `paste_retain_style_properties` [docs here](https://www.tiny.cloud/docs/plugins/paste/#paste_retain_style_properties). 

The difference in behaviours between the paste plugin and browser paste comes from the paste plugin attempting to sanitise content as it enters the editor. If you're interested, have a look at the source code view after pasting content from google docs into the editor with and without the paste plugin.

**Comment by niuage** [comment]
Thank you, I guess I missed this option!

Have you tried your solution in the Codepen above? https://codepen.io/niuage/pen/MWyVEpo

I updated it, and added `paste_retain_style_properties: 'all',` to the config. When copying the content of [this doc](https://docs.google.com/document/d/1mi93gyNwNnEKURw-pPtoN15OQJPI5ZqgjRW12uPl8zo/edit) into it, it still transforms the list into a numbered one. Am I missing something obvious?

**Comment by ashfordneil** [comment]
I tried it in the codepen you linked. What I needed to do to make it work was remove the `extended_valid_elements` section of the config. I've copied the config from your codepen across to [this fiddle](https://fiddle.tiny.cloud/fthaab) and then made the changes necessary to make the paste work. I was able to preserve the numbering style when pasting (from chrome / firefox / safari and to chrome / firefox / safari) without losing the formatting. Let me know how you go with the fiddle.

**Comment by niuage** [comment]
Thanks to both of you. Honestly I had forgotten I had `extended_valid_elements ` in my config, that was part of the things I tried when experimenting and forgot to remove it.

I think the issue is ready to be closed 👍
